.github/workflows/scripts/reviewReviewed.js

```diff
@@ -41,20 +41,20 @@ const messageAssignee = async ({ context }) => {
     try {
```

Comment:
While these changes are minor, we don't want to change anything in the `.github` folder, so let's revert these changes. These are from the prettier formatting, so remember to add the `.prettierignore` file as mentioned in the pnp-dev slack channel [here](https://ctcuci.slack.com/archives/C042J7C48NL/p1668647446041759?thread_ts=1668210489.115029&cid=C042J7C48NL).